apps/google-analytics-4/frontend/src/components/config-screen/GoogleAnalyticsConfigPage.tsx

```diff
@@ -125,8 +125,20 @@ export default function GoogleAnalyticsConfigPage() {
       return false;
     }
 
+    // Remove empty content type rows
+    let parametersToSave = parameters;
+
+    if (parameters.contentTypes) {
+      const filteredContentTypes = Object.fromEntries(
```

Comment:
I had to look at this a minute to figure out what you were filtering and then guess why. Maybe we could just make it easier to understand with a more precise label such as `nonEmptyContentTypes`? not sure a comment is needed but could help (maybe something like "// we just want to filter out any empty content types that came from empty rows on the from").

Reply:
Yes! I will update this for clarity